packages/server/modules/fileuploads/domain/events.ts

```diff
@@ -10,10 +13,7 @@ export const FileuploadEvents = {
 
 export type FileuploadEvents = (typeof FileuploadEvents)[keyof typeof FileuploadEvents]
 
-type FileuploadStartedPayload = Pick<
-  FileUploadRecordV2,
-  'userId' | 'projectId' | 'fileSize' | 'fileType'
->
+type FileuploadStartedPayload = { upload: FileUploadRecordV2 & FileUploadRecord }
```

Comment:
@danielgak 

1. If the target entity is set as the entire event payload, then when you have to add new props you have to pollute the target entity type (e.g. `type CreatedPayload = FileUploadRecord & { branchCreatedAt: Date }`
2. If we're already retrieving the entire FileUpload entity, then preferably we should just emit the entire record instead of only a few of the props (for easier reuse)

Reply:
also - we have values for both v1 and v2 record available, so lets emit both for easier handling in listeners